modules/options/src/main/scala/scala/build/actionable/ActionableHandler.scala

```diff
@@ -0,0 +1,29 @@
+package scala.build.actionable
+import scala.build.Positioned
+import scala.build.errors.BuildException
+import scala.build.options.BuildOptions
+
+trait ActionableHandler[V] {
+
```

Comment:
NIT Maybe the type param could be changed to a type member, as it's only used to pass things around its various methods, like
```suggestion
trait ActionableHandler {
  type V
```

Things ought to compile down-the-line with this.